### ISSUE TITLE
trezorctl: common exception handling

### ISSUE THREAD

**Issue by matejcik** [trezor/python-trezor#271]
Currently, every command has to handle its own exceptions. So most canceled actions throw a stack trace, unless specifically coded against.

That is stupid.

We should have a common exception handler for the subcommands and output nice messages whenever it makes sense.

Also maybe stop for a minute and rethink raising exceptions when an action is canceled?

**Comment by cculianu** [comment]
Yeah I agree. It's terrible the way it is. Electrum and Electron Cash both have horrible UX's as a result.

**Comment by matejcik** [comment]
yeah no, that's completely unrelated. these apps are using trezor as a library and can do their own exception handling, so if there's terrible UX that's their problem

this issue is specifically about bad exception handling in `trezorctl` the CLI tool

**Comment by cculianu** [comment]
Ok, I'll shut up now. :)
